v2/emailpassword/common-customizations/sessions/session-verification-in-api/verify-session.mdx

```diff
@@ -598,6 +619,27 @@ export default async function likeComment(req: any, res: any) {
 }
 ```
 
+</TabItem>
+<TabItem value="nextjs-app">
+
+```tsx
+import { NextResponse, NextRequest } from "next/server";
+import SuperTokens from "supertokens-node";
+import { withSession } from "supertokens-node/nextjs";
+// @ts-ignore
+import { backendConfig } from "@/app/config/backend";
+
+SuperTokens.init(backendConfig());
+
+export function POST(request: NextRequest) {
+    return withSession(request, async (session) => {
```

Comment:
you missed passing in sessionRequired: false, and handling the case when session is undefined. Please double check your code snippets.

Reply:
Fixed it! I checked other snippets once again, couldn't find anything else.